sound/soc/sof/intel/mtl.c

```diff
@@ -18,6 +18,7 @@
 #include "hda-ipc.h"
 #include "../sof-audio.h"
 #include "mtl.h"
+#include "telemetry.h"
```

Comment:
you should clarify in the commit message that this also adds support for telemetry on LNL since the mtl code is reused.

Reply:
sure, updated. Thanks!